binder/views.py

```diff
@@ -751,24 +745,28 @@ def _get_with_ids(self, pks, request, with_map, where_map):
 	# should be scoped on, but also a set of wheres where model M
 	# should further be scoped on.
 	#
-	# Returns a queryset for this field we should filter on further.
-	def _filter_relation(self, M, where_map):
+	# Returns a Q object for this field we should filter on further.
```

Comment:
It now returns a tuple. Miight be wise to describe also the second part of the tuple, as this is probably the first place people will look to understand what `_filter_relation` is returning

Reply:
Yeah, I added that later, and don't really consider this to be something it should do, but couldn't find a way around. Good idea to update the comment to match.